src/main/java/uk/gov/companieshouse/presenter/account/model/PresenterAccountAddress.java

```diff
@@ -0,0 +1,206 @@
+package uk.gov.companieshouse.presenter.account.model;
+
+import java.util.Optional;
+
+import org.springframework.data.mongodb.core.mapping.Field;
+
+import uk.gov.companieshouse.presenter.account.exceptionhandler.ValidationException;
+import uk.gov.companieshouse.presenter.account.model.request.presenter.account.PresenterAddress;
+import uk.gov.companieshouse.presenter.account.validation.utils.StringValidator;
+
+public class PresenterAccountAddress {
+
+    private static final int PREMISES_MAX_LENGTH = 40;
+    private static final int LINE_1_MAX_LENGTH = 40;
+    private static final int LINE_2_MAX_LENGTH = 40;
+    private static final int COUNTY_MAX_LENGTH = 40;
+    private static final int COUNTRY_MAX_LENGTH = 40;
+    private static final int POSTCODE_MAX_LENGTH = 10;
+    
+    @Field
+    private final String premises;
+
+    @Field
+    private final String addressLine1;
+
+    @Field
+    private final String addressLine2;
+
+    @Field
+    private final String county;
+
+    @Field
+    private final String country;
+
+    @Field("postCode")
+    private final String postcode;
+
+    public static PresenterAccountAddress createPresenterAccountAddress(PresenterAddress presenterAddress){
+        if (presenterAddress != null) {
+            return new PresenterAccountAddress(presenterAddress.premises(), 
+            presenterAddress.addressLine1(), 
+            presenterAddress.addressLine2(), 
+            presenterAddress.county(), 
+            presenterAddress.country(), 
+            presenterAddress.postcode());
+        }
+        else{
```

Comment:
Formatting a bit off here
```suggestion
        } else {
```